### DISCUSSION TITLE
Is volt_ac_SPICE.cpp (Vac_SPICE) deprecated?

### DISCUSSION THREAD

**Discussion by iwbnwif** [ra3xdh/qucs_s#550] [Q&A] [answered]
I am trying to understand the difference between Vac_SPICE and S4Q_V.

Both seem to allow ngnutmeg equations to be entered as parameters with multiple lines allowed. However, I cannot find where VacSPICE can be selected in the components palette, so I wonder if it is still used?

**Comment by ra3xdh** [comment, marked as answer]
Yes, the Volt_AC source was removed in recent release. Its appearance and functions duplicate the generic SPICE V source. I cannot say why the Volt_AC source was designed in 2015.

**Reply by iwbnwif** [reply]
Many thanks, that is clear.
